### ISSUE TITLE
Website updates not being deployed

### ISSUE THREAD

**Issue by stephenroller** [facebookresearch/ParlAI#2203]
The deploy_website step is failed on master, but nothing has changed. Could be just flakiness of some upstream servers, or perhaps debian changed or something?

From [CircleCI](https://circleci.com/gh/facebookresearch/ParlAI/19265?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link):

```
#!/bin/bash -eo pipefail
sudo apt-get install s3cmd
s3cmd --access_key="${S3_ACCESS_KEY}" --secret_key="${S3_SECRET_KEY}" sync -f --delete-removed website/build/ "s3://parl.ai/"
s3cmd --access_key="${S3_ACCESS_KEY}" --secret_key="${S3_SECRET_KEY}" setacl --acl-public --recursive "s3://parl.ai/"
s3cmd --access_key="${S3_ACCESS_KEY}" --secret_key="${S3_SECRET_KEY}" modify --add-header="Content-type:text/css" 's3://parl.ai/static/css/*' 's3://parl.ai/docs/_static/*.css' 's3://parl.ai/docs/_static/css/*.css'
Reading package lists... Done


Building dependency tree       


Reading state information... Done

The following additional packages will be installed:
  python-dateutil python-magic
The following NEW packages will be installed:
  python-dateutil python-magic s3cmd
0 upgraded, 3 newly installed, 0 to remove and 7 not upgraded.
Need to get 194 kB of archives.
After this operation, 736 kB of additional disk space will be used.

Get:1 http://deb.debian.org/debian stretch/main amd64 python-dateutil all 2.5.3-2 [51.3 kB]

Err:2 http://deb.debian.org/debian stretch/main amd64 python-magic all 1:5.30-1+deb9u1
  404  Not Found

Get:3 http://deb.debian.org/debian stretch/main amd64 s3cmd all 1.6.1-1 [94.3 kB]

Err:2 http://security.debian.org/debian-security stretch/updates/main amd64 python-magic all 1:5.30-1+deb9u1
  404  Not Found

Fetched 146 kB in 0s (434 kB/s)
E: Failed to fetch http://security.debian.org/debian-security/pool/updates/main/f/file/python-magic_5.30-1+deb9u1_all.deb  404  Not Found
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
Exited with code 100
```

**Comment by stephenroller** [comment]
GDI that didn't fix it, how?!

https://linuxconfig.org/installation-of-amazon-s3cmd-command-line-s3-tool-on-debian

**Comment by stephenroller** [comment]
Fixed in #2217.
